guides/integrate-nextauth.md

```diff
@@ -0,0 +1,121 @@
+
+**Application**
+
+![application-config](/assets/nextauth-application-config.png)
+
+**Authenticator**
+
+![authenticator-config](/assets/nextauth-authenticator-config.png)
+
+### Create an Identity in the BI console
+
+We will create identities that can login to the NextAuth site here, and enroll them with a BI credential bound to their device.
+
+1. In the BI Admin console, click **"Identities"**
+1. If the user you wish to use already exists, skip to step 5.
+1. Click **"Add identity"**
+1. Input the desired Name, Username, and Email (All three are required)
+1. Next, we will send the user an enrollment email to bind a credential to their device.
+1. Follow the steps at [Send Enrollment Emails](send-enrollment) for each new identity. In the near future, this will become a push-button operation, but for now it involves sending commands via CURL.
+1. Each new identity will receive an Enrollment email, which they click on to bind a credential to their device (desktop, laptop, mobile, etc).
+
+## Configure NextAuth for OIDC
+
+### Install the free Plugin
+
+This guide is based on the popular [Available OAuth providers](https://authjs.dev/reference/providers/oauth-builtin) which is regularly updated and has many thousands of Active installations.
+
+Follow the installation steps specified in the [Beyond Identity](https://authjs.dev/reference/oauth-providers/auth0) Guide
+
+### Configure the plugin
+
+These steps will help you configure the plugin to use with Beyond Identity.
+
+1. Keep the Beyond Identity Admin Console open in a tab or window to copy and paste values in
+1. Create three Environment Variables: BEYOND_IDENTITY_CLIENT_ID, BEYOND_IDENTITY_CLIENT_SECRET, BEYOND_IDENTITY_ISSUER
+1. For **"BEYOND_IDENTITY_CLIENT_ID"**, copy and paste the value from Applications -> Your New Application -> External Protocol -> Client ID
+1. For **"BEYOND_IDENTITY_CLIENT_SECRET"**, copy and paste the value from Applications -> Your New Application -> External Protocol -> Client Secret
+1. For **"BEYOND_IDENTITY_ISSUER"**, copy and paste the value from Applications -> Your New Application -> External Protocol -> Issuer
+
+### Add the redirect URL to the BI console
+
+The Beyond Identity web authenticator needs to know where to redirect the user after a successful authentication.
+
+1. On the BI Admin console, under Applications -> Your new application -> Redirect URIs, set the Redirect URL
+    - If you do not know your redirect, this can be found by clicking on the "Sign in with Beyond Identity" button, checking the url, and decoding the "redirect_uri" parameter.
+1. Hit **"Submit"** at the bottom of the page.
+
+Congratulations, you have configured the BI console and the OIDC client plugin.
+
+### Try logging in
+
+We will now attempt to log in and verify successful authentication.
+
+1. Using a device where you have [created an identity and then enrolled a credential](#Create an Identity in the BI console), Visit your app or website to login
```

Comment:
On the preview we see the link that looks like this:

`Using a device where you have [created an identity and then enrolled a credential](#Create an Identity in the BI console), Visit your app or website to login`

Reply:
Thanks! This seems to be broken in the wordpress guide as well. I'll fix both.